central/complianceoperator/v2/compliancemanager/manager_impl.go

```diff
@@ -54,23 +87,152 @@ func (m *managerImpl) ProcessComplianceOperatorInfo(ctx context.Context, complia
 	return m.integrationDS.UpdateComplianceIntegration(ctx, complianceIntegration)
 }
 
-func (m *managerImpl) ProcessScanRequest(_ context.Context, _ interface{}) error {
-	//TODO:
-	// 1. Upsert to database
-	// 2. Push request to Sensor
-	panic("implement me")
+// ProcessScanRequest processes a request to apply a compliance scan configuration to one or more Sensors.
+func (m *managerImpl) ProcessScanRequest(ctx context.Context, scanRequest *storage.ComplianceOperatorScanConfigurationV2, clusters []string) (*storage.ComplianceOperatorScanConfigurationV2, error) {
+	if !features.ComplianceEnhancements.Enabled() {
+		return nil, nil
+	}
+
+	// Convert and validate schedule
+	// TODO(ROX-19818):  remove default schedule
+	// For MVP a schedule is required.  If one is not present use daily at midnight.  As it evolves
+	// it may be as simple as no schedule means a one time scan.
+	cron := defaultScanSchedule
+	var err error
+	if scanRequest.GetSchedule() != nil {
+		cron, err = schedule.ConvertToCronTab(scanRequest.GetSchedule())
+		if err != nil {
+			log.Error(err)
+			return nil, errors.Errorf("Unable to convert schedule for scan configuration named %q to cron.", scanRequest.GetScanName())
```

Comment:
The `err` from conversion while include insights to fix the problem. WIthout that, it could be hard for the users to figure out the problem.

```suggestion
			err =  errors.Wrapf(err, "Unable to convert schedule for scan configuration named %q to cron.", scanRequest.GetScanName())
			log.Error(err)
			return nil, err
```